pkg/agent/plugin/svidstore/svidstore.go

```diff
@@ -43,4 +43,7 @@ type SVID struct {
 
 	// Expiration timestamp
 	ExpiresAt time.Time
+
+	// Hint is used for indicating the svid usage
+	Hint string
```

Comment:
What do you think about move hint to be a property of a secret (like spire property we are already storing) 
that will allow users to see hints of a secret before consuming it (that will require to get secret object and parse) 
That will allow uses to filter by hint too